### PR TITLE
feat: add sponsorship logo

### LINKED ISSUE #70: Add Sponsorship Logo
untuk logo sponsor yang uda tandatangan MoA, sudah bisa ditampilin di webnya..
ini keknya aku yang bakalan ngerjain v: buat hacktoberfest

### PR DESCRIPTION
**Pull request by gelembungsahabat**
resolve #70 
- add logo sponsor
- change size logo

### CODE REVIEW

**Comment by gelembungsahabat** [comment]
@ridhof up

File: components/global/Sponsor.vue [hunk 1]
```diff
@@ -67,15 +67,15 @@ export default {
...
         ],
         gold: [
-          // { image: 'https://2020.pycon.id/theme/img/supporters/logo-scoutapm.png', slug: '' }
-          // { image: 'https://2020.pycon.id/theme/img/supporters/logo-rwid.jpg', slug: '' },
-          // { image: 'https://2020.pycon.id/theme/img/supporters/logo-vonage.png', slug: '' }
+          { image: 'https://ik.imagekit.io/gelembungsahabat/shopee_logo_en_e3hmbkpyN.png?updatedAt=1635638155632', slug: '' }
         ],
         silver: [
-          // { image: 'https://2020.pycon.id/theme/img/supporters/logo-qiscus.png', slug: '' }
+          { image: 'https://ik.imagekit.io/gelembungsahabat/logo-warna__1__wa1MFvQdX.png?updatedAt=1635181524902', slug: '' }
```
Comment (ridhof) [inline components/global/Sponsor.vue:78]: would be nice if gold and silver sponsors got a link to their own website so other people can check their company right away

File: components/global/Sponsor.vue [hunk 2]
```diff
@@ -67,15 +67,15 @@ export default {
       sponsors: {
         show: true,
         platinum: [
-          // { image: 'https://2020.pycon.id/theme/img/supporters/logo-microsoft.png', slug: '' }
+          { image: 'https://ik.imagekit.io/gelembungsahabat/AWS_logo_RGB_sXBNduNmi.png?updatedAt=1635181586293', slug: '' },
```
Comment (ridhof) [inline components/global/Sponsor.vue:70]: I don't think updatedAt as a query param is necessary on imagekit, we should remove it in the future for cleaner code

**Review by ridhof — APPROVED** [review]
LGTM!

File: components/global/Sponsor.vue (hunk 1 above)
Comment (ridhof) [reply components/global/Sponsor.vue:78]: added a new issue regarding sponsor link #76

File: components/global/Sponsor.vue (hunk 2 above)
Comment (ridhof) [reply components/global/Sponsor.vue:70]: added a new issue regarding updatedAt query param for imagekit #77

**Pull request merged**